app/helpers/application_helper.rb

```diff
@@ -34,6 +34,13 @@ def date_string(start_date, end_date)
     result
   end
 
+  # Returns the time with the conference's timezone
+  def set_offset(time, timezone)
```

Comment:
@Ana06  We can use the parse method but as we will be converting the time into the ISO 8601 format displaying the timezone name isn't required like in the first method.